UnityProject/Assets/Scripts/UI/Jukebox/GUI_Jukebox.cs

```diff
@@ -0,0 +1,60 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+public class GUI_Jukebox : NetTab
+{
+	[SerializeField]
+	private NetLabel labelSong;
+
+	[SerializeField]
+	private NetLabel labelArtist;
+
+	[SerializeField]
+	private NetLabel labelTrack;
+
+	[SerializeField]
+	private NetPrefabImage prefabImagePlayStop;
+
+	private Jukebox _JukeboxController;
+	private Jukebox jukeboxController
+	{
+		get
+		{
+			if (_JukeboxController == null)
+				_JukeboxController = Provider.GetComponent<Jukebox>();
+
+			return _JukeboxController;
+		}
+	}
+
+	public void OnTabOpenedHandler(ConnectedPlayer connectedPlayer)
+	{
+		labelTrack.Value = jukeboxController.TrackPosition;
+		labelSong.Value = jukeboxController.SongName;
+		labelArtist.Value = jukeboxController.Artist;
+		prefabImagePlayStop.Value = jukeboxController.PlayStopButtonPrefabImage;
+	}
+
+	public void PlayOrStop()
+    {
+		if (jukeboxController.IsPlaying)
```

Comment:
`jukeboxController.IsPlaying ? jukeboxController.Stop() : jukeboxController.Play();`

Reply:
That won't be legal C#.  I was surprised of it too.  Apparently, you just can't use the conditional operator on it's own without assigning to something in return or piping the result to a function.

int a = 5;
string b = (a == 5) ? a.ToString() : 666.ToString();  // This is legal
Console.WriteLine((a == 5) ? a.ToString() : 666.ToString(); // This is legal
a == 5 ? a.ToString() : 666.ToString(); // This isn't legal (And it doesn't even make sense...)